applications/io_board/targets/freertos.armv6m.st-stm32g0b1re-nucleo/main.cxx

```diff
@@ -0,0 +1,148 @@
+/** \copyright
+ * Copyright (c) 2013, Balazs Racz
```

Comment:
Should the dates be updated?

Reply:
I don't think so, since this file was forked without any modification.